packages/fetchai/protocols/fipa/fipa.proto

```diff
@@ -15,6 +15,7 @@ message FIPAMessage{
     }
     message Propose{
         repeated bytes proposal = 1;
+        bytes tx_nonce=2;
```

Comment:
no, the tx_nonce should be part of the proposal itself in my opinion. so no change to proto file needed